tests/python/contrib/test_vitis_ai/test_vitis_ai_codegen.py

```diff
@@ -241,6 +240,13 @@ def test_upsampling(dpu_target):
     verify_codegen(mod, dpu_target=dpu_target)
 
 
+@pytest.mark.skip(
+    reason="I and O used to be mixed up in kernel layouts in TVM."
+    "This is fixed, but vitis needs to adopt the new convention."
+    "To change, simply remove this line:"
+    "https://github.com/Xilinx/pyxir/blob/bef661d6d77adcdbd2cf4163f2cf3a1d31d40406/"
+    "python/pyxir/frontend/tvm/relay_tools/relay_l2_convolution.py#L380"
+)
```

Comment:
cc @jtuyls

Reply:
Thanks, this kernel layout switch makes a lot of sense to me. I will fix this in the Vitis flow.